services/ui/serverless.yml

```diff
@@ -82,6 +93,7 @@ resources:
     CloudFrontWebAcl:
       Type: AWS::WAFv2::WebACL
       Properties:
+        Name: ${self:custom.webAclName}
```

Comment:
This is actually kind of important.  The webacl name is required to look up its ARN... the ARN is required to make the association with firehose in the post deploy hook.  By setting the name, I know what to look for.